fairroot/base/sim/FairMCApplication.h

```diff
@@ -126,9 +127,7 @@ class FairMCApplication : public TVirtualMCApplication
     TChain* GetChain();
     /** Initialize geometry
         Called by TVirtualMC.
-        TGeant3 calls AddIons() first, then InitGeometry().
-        TGeant4 calls InitGeometry() first, then AddIons().
-        This function also registers detectors.*/
+        This function, if called after AddIons(), calls InitFinalizer()*/
```

Comment:
1. I liked the old notices how Geant3 / Geant4 work together.
2. In your current design it does not call InitFinalizer.

Reply:
Agree, done.